diff --git a/EventFilter/L1TRawToDigi/plugins/MP7BufferDumpToRaw.cc b/EventFilter/L1TRawToDigi/plugins/MP7BufferDumpToRaw.cc
index 7bbec01..e94fe96 100644
--- a/EventFilter/L1TRawToDigi/plugins/MP7BufferDumpToRaw.cc
+++ b/EventFilter/L1TRawToDigi/plugins/MP7BufferDumpToRaw.cc
@@ -395,6 +395,7 @@ MP7BufferDumpToRaw::formatAMC(amc13::Packet& amc13, const std::vector<Block>& bl
   // TODO this is an empty word to be replaced with a proper MP7
   // header containing at least the firmware version
   load32.push_back(0);
+  load32.push_back(0);
   for (const auto& block: blocks) {
     LogDebug("L1T") << "Adding block " << block.header().getID() << " with size " << block.payload().size();
     auto load = block.payload();
diff --git a/EventFilter/L1TRawToDigi/python/stage1MP7BufferRaw_cfi.py b/EventFilter/L1TRawToDigi/python/stage1MP7BufferRaw_cfi.py
index bdf8ba7..cd55f42 100644
--- a/EventFilter/L1TRawToDigi/python/stage1MP7BufferRaw_cfi.py
+++ b/EventFilter/L1TRawToDigi/python/stage1MP7BufferRaw_cfi.py
@@ -10,11 +10,11 @@ stage1Raw = cms.EDProducer(
 
     # parameters for non-packetised data
     nFramesPerEvent  = cms.untracked.int32(6),
-    nFramesOffset    = cms.untracked.vint32(0,9),
-    nFramesLatency   = cms.untracked.vint32(0,0),
+    nFramesOffset    = cms.untracked.vint32(0),
+    nFramesLatency   = cms.untracked.vint32(0),
 
     # DAQ parameters
-    boardId          = cms.untracked.vint32( 0x100D, 0x100E ),  # jet ID, EG ID
+    boardId          = cms.untracked.vint32(0x2301),  # jet ID, EG ID
     fedId            = cms.untracked.int32(1352),
     eventType        = cms.untracked.int32(238),
     fwVersion        = cms.untracked.int32(255),
@@ -58,36 +58,5 @@ stage1Raw = cms.EDProducer(
                                                     2,2,2,2, # quad 12
                                                     2,2,0,0) # quad 13
             ),
-        cms.untracked.PSet(  # EG board setup
-            rxBlockLength    = cms.untracked.vint32(6,6,6,6, # quad 0
-                                                    6,6,6,6, # quad 1
-                                                    6,6,6,6, # quad 2
-                                                    6,6,6,6, # quad 3
-                                                    6,6,6,6, # quad 4
-                                                    6,6,6,6, # quad 5
-                                                    6,6,6,6, # quad 6
-                                                    6,6,6,6, # quad 7
-                                                    6,6,6,6, # quad 8
-                                                    0,0,0,0, # quad 9
-                                                    0,0,0,0, # quad 10
-                                                    0,0,0,0, # quad 11
-                                                    0,0,0,0, # quad 12
-                                                    0,0,0,0) # quad 13
-            ,
-            txBlockLength    = cms.untracked.vint32(0,0,0,0, # quad 0
-                                                    0,0,0,0, # quad 1
-                                                    0,0,0,0, # quad 2
-                                                    0,0,0,0, # quad 3
-                                                    0,0,0,0, # quad 4
-                                                    0,0,0,0, # quad 5
-                                                    0,0,0,0, # quad 6
-                                                    0,0,0,0, # quad 7
-                                                    0,0,0,0, # quad 8
-                                                    0,0,0,0, # quad 9
-                                                    0,0,0,0, # quad 10
-                                                    0,0,0,0, # quad 11
-                                                    0,0,0,0, # quad 12
-                                                    0,0,0,0) # quad 13
-            )
-        )
-)
+        ),
+    )
diff --git a/EventFilter/L1TRawToDigi/src/implementations_stage1/CaloSetup.cc b/EventFilter/L1TRawToDigi/src/implementations_stage1/CaloSetup.cc
index 56555af..06ace63 100644
--- a/EventFilter/L1TRawToDigi/src/implementations_stage1/CaloSetup.cc
+++ b/EventFilter/L1TRawToDigi/src/implementations_stage1/CaloSetup.cc
@@ -114,9 +114,10 @@ namespace l1t {
                      auto rct_unp = UnpackerFactory::get()->make("stage1::RCTEmRegionUnpacker");
 
                      // 4109 == 0x100D
-                     if(board == 0x2300){  
+                     if(board == 0x2301){  
                         res[77] = cjet_unp_Left;
                         res[79] = cjet_unp_Right;  
+                        /*
                         res[81] = fjet_unp_Left;
                         res[83] = fjet_unp_Right;
                         res[85] = iegamma_unp_Left;
@@ -131,6 +132,7 @@ namespace l1t {
                         res[103] = tau_unp_Right;
                         res[105] = isotau_unp_Left;
                         res[107] = isotau_unp_Right;
+                        */
 
                         for (int m=0;m<36;m++) {
                            res[m*2] = rct_unp;
